docs/source/conf.py

```diff
@@ -22,6 +22,13 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #sys.path.insert(0, os.path.abspath('.'))
```

Comment:
If you uncomment this line and change `.` to `../..` (i.e. so it points to the repo root), the notebook modules should be importable even though it fails to install. It's a bit of a hack. I'd forgotten that the notebook can't be installed from the repo without npm.